helm/kubernetes-coredns-chart/templates/configmap.yaml

```diff
@@ -31,4 +31,5 @@ data:
         {{- end }}
```

Comment:
Do i also need to bump versions?

Reply:
No breaking changes, no need to bump version IMO.